ChangeLog.md

```diff
@@ -15,6 +15,8 @@ full changeset diff at the end of each section.
 
 Current Trunk
 -------------
+ - Error if `_main` is missing by default.  `_main` has alwasy been in the
+   default EXPORTED_FUNCTIONS by previously did not generate an error.
```

Comment:
Please also mention how to fix this if users hit an error, by specifying EXPORTED_FUNCTIONS